msusicky/ockovani-covid
# New models for UZIS data, migrations, app refactoring

closes #56, #57, #58

TODO:
- [x] test/fix fetcher
- [x] configure flask to run on 0.0.0.0
- [x] create google table with additional ids
- [x] add fetcher for the google table 
- [x] fix views after the last refactoring (VolnaMistaRaw->VolnaMistaDen, VolnaMista->VolnaMistaCas)
- [x] set log level to info
- [x] implement API check in google table
- [x] fill google table
- [x] fix CSV exports

Upgrade guide:
- drop all tables in database (or create new database)
- check connection string in config.py
- install new requirements
- execute database migrations: `flask db upgrade` 
- configure fetchers
  - fill config.ini according to the template
  - create google_key.json in the root directory (documented here: https://gspread.readthedocs.io/en/latest/oauth2.html)
- execute fetchers: `flask fetch-all`
- execute webserver `flask run --host 0.0.0.0`
- publish is without changes
